snap/info.go

```diff
@@ -253,8 +253,15 @@ type ChannelSnapInfo struct {
 	Size        int64           `json:"size"`
 }
 
-// Name returns the blessed name for the snap.
-func (s *Info) Name() string {
+// InstanceName returns the blessed name of the snap decorated with instance
+// key, if any.
+func (s *Info) InstanceName() string {
+	// TODO parallel-install: include instance key
+	return s.StoreName()
+}
+
+// StoreName returns the name of the snap without the instance key.
```

Comment:
StoreName returns the global blessed name of the snap. 

or something like that